experiments/functional/jiva-volume-resize/test.yml

```diff
@@ -0,0 +1,328 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+          ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+          ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+        
+        - name: Update the DaemonSet template with the variables.
+          template:
+            src: daemonset.j2
+            dest: daemonset.yml
+
+        - name: Create the DaemonSet 
+          shell: kubectl apply -f daemonset.yml
+          args:
+            executable: /bin/bash
+          register: status
+          failed_when: "status.rc != 0"
+
```

Comment:
Include a task to check the daemonset is running

Reply:
Added the required task to check daemonset is running